frontend/public/index.html

```diff
@@ -63,6 +63,8 @@
       name="twitter:image"
       content="%PUBLIC_URL%/static/images/__OG_IMAGE__"
     />
+    <!-- Load the datadog js chunk before the app -->
+    <script src="./static/js/datadog-chunk.js"></script>
```

Comment:
Considering we do have some variables in the file (e.g. capture sample rate), and some `beforeSend` code to alter behaviour (which could be modified in the future), we need to make sure the file is not unduly cached by clients or CF.

We could inject the file content hash into its name, like we do for the other chunks, or make sure the file is served with `no-cache` (or low time cache).

Reply:
hash added in [1bdeeb6](https://github.com/opengovsg/FormSG/pull/5571/commits/1bdeeb6070a6307992c0bb28d16b2be3e4919ed6)